pkgs/top-level/all-packages.nix

```
@@ -3006,6 +3006,12 @@ with pkgs;
 
   pikchr = callPackage ../tools/graphics/pikchr { };
 
+  poop = callPackage ../tools/misc/poop {
+    zigHook = zigHook.override {
+      zig = zig_0_11;
```

```suggestion
      zig = buildPackages.zig_0_11;
```

Cross-compilation magic.